pkg/iris/api.go

```diff
@@ -32,20 +37,21 @@ func NewApi(client *http.Client, kind string, uri string) API {
 		client: client,
 		kind:   kind,
 		uri:    fmt.Sprintf("%s%s", protocol, uri),
+		token:  token,
 	}
 }
 
-func (a *api) GetConfiguration(configurationName string, accessToken string) ([]byte, error) {
-	configUrl := fmt.Sprintf("%s/services/vsm-iris/v1/configurations/kubernetes/%s", a.uri, configurationName)
-	req, err := http.NewRequest("GET", configUrl, nil)
+func (a *api) GetConfiguration(configurationName string) ([]byte, error) {
 	if configurationName == "" {
 		return nil, errors.New("configuration name should not be null or empty")
 	}
+	configUrl := fmt.Sprintf("%s/services/vsm-iris/v1/configurations/kubernetes/%s", a.uri, configurationName)
```

Comment:
Maybe we should change the controller in iris so it requires the workspace id?

Reply:
Yes they are. The workspace id should be fetched from the token that will be passed to iris.

The token is a technical user in that specific customer workspace so the token will include the workspace id.